pkg/util/version/const.go

```diff
@@ -25,6 +25,14 @@ const (
 
 var GitCommit = "unknown"
 
+// InstallerImageDigest is the mapping of a minor version to the aro-installer wrapper digest
+// this allows us to utilize SDP instead of pushing up the tag and it rolling out to all regions
+// at once
+var InstallerImageDigest = map[string]string{
+	NewVersion(4, 10).MinorVersion(): "eef5f0d82ab07c866999f99a632edafd845e0dc48a0f06eac9df46b0ab882231",
+	NewVersion(4, 11).MinorVersion(): "82869dd10841046c4b98fc46c6f87030c82b320b82f990610c1fa87150004730",
```

Comment:
I believe these are the current SHA values of the installer images. How do we intend to manage these (and the version numbers in the future?) If we have to make an update here every time we want to rollout a new installer image we'd be linking the ARO-Installer code work directly to RP work deploy. Any way to interpret these values?

Reply:
We're going to move them into the RP config.  https://github.com/Azure/ARO-RP/pull/2967#discussion_r1231676683